src/firefly/js/ui/UploadTableChooser.js

```diff
@@ -82,23 +89,88 @@ function uploadSubmit(request, setUploadInfo)  {
+    const tables = getTableGroup().tables;
+    const idx = tbl.highlightedRow;
+    const activeTblId = get(tables, [Object.keys(tables)[idx], 'tbl_id']);
+    const activeTbl = getTblById(activeTblId);
+    const tableRequest = activeTbl.request;
+    const columnData = activeTbl.tableData.columns;
+    //ToFix: columns have entries ROW_IDX and ROW_NUM which cause error if not removed
+    //should columns also be ordered the same as existing table?
+    const columns = columnData.filter((elem) => elem.name !== 'ROW_IDX' && elem.name !== 'ROW_NUM');
+
+    const params = omitBy({
+        [ServerParams.COMMAND]: ServerParams.TABLE_SAVE,
+        [ServerParams.REQUEST]: JSON.stringify(tableRequest),
+        file_name: tableRequest.META_INFO.title,
+        file_format: Format.IPACTABLE,
+        table_save: 'true'
+    }, isNil);
+
+    doJsonRequest(ServerParams.TABLE_SAVE, params).then((result) => {
+        if (!result.success) {
+            showInfoPopup('Error loading this table', result.error);
+            return false;
+        }
+        const uploadInfo = {
+            serverFile: result?.serverFile ?? null,
+            title: activeTbl.title,
+            fileName: activeTbl.title,
+            tbl_id: activeTblId,
+            columns: columns,
+            totalRows: activeTbl.totalRows,
+            tableSource: EXISTING_TBL_SOURCE,
+        };
+        setUploadInfo(uploadInfo);
+        dispatchHideDialog(dialogId);
+    });
     return false;
 }
 
-
+const LoadedTables= (props) => {
+    const {onSubmit, onCancel=dispatchHideDropDown, keepState=true, groupKey} = props;
+    const tables = getTableGroup()?.tables ?? null;
+    if (!tables) {
+        return (
+            <div style={{color: 'gray', margin: '10px', fontSize: 'larger', lineHeight: '1.3em', padding: '5px 0 5px 0'}}>
+                {'No tables available to load.'}
+            </div>
+        );
+    }
+    let data = []; //table titles
+    for (const tbl in tables) {
+        const title = [tables[tbl].title];
+        data.push(title);
+    }
+    const columns = [{name: 'Loaded Tables', width: 80}];
+    const highlightedRow = 0; //default selection
+    const tbl_id = 'existing-table-list';
+
+    const tableModel = {tbl_id, tableData: {columns, data}, highlightedRow, totalRows: data.length};
+
+    return (
+        <div style={{margin: '10px', position: 'relative', width: '100%', display: 'flex', alignItems: 'stretch', flexDirection: 'column'}}>
+            <div style={{color: 'gray', fontSize: 'larger', lineHeight: '1.3em'}}>
+                {'Select one of the existing tables below to load into the TAP panel: '}
+            </div>
+            <FieldGroup groupKey={groupKey} keepState={keepState} style={{height:'100%', width: '100%',
+                display: 'flex', alignItems: 'stretch', flexDirection: 'column'}}>
+                <FormPanel
+                    width='auto' height='auto' onSubmit={onSubmit}
```

Comment:
I don't think there's `width` and `height` props in TablePanel.

Reply:
You mean FormPanel right @loitly? thanks, fixed this here and also in FileUploadDropdown.jsx.